hw/ip/otbn/data/otbn.hjson

```diff
@@ -125,6 +125,7 @@
 
             - 0x0 (ErrCodeNoError):     No error occurred.
             - 0x1 (ErrCodeBadDataAddr): Load or store to invalid address
+            - 0x2 (ErrCodeCallStack):   Call stack underflow/overflow
```

Comment:
Can you please file an issue (or do the code change) to get this error code added to the DIF?

Reply:
Done, I think. Could you take a quick look to make sure I haven't done anything stupid?

Reply:
Looks good, thanks!